website/docs/disaster_recovery.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Disaster and Recovery with Apache Hudi
+toc: true
+---
+
+Disaster and Recovery is very much mission critical for any software. Especially when it comes to data systems, the impact could be very serious
+leading to delay in business decisions or even wrong business decisions at times. So, Apache Hudi has built in features to assist 
+you in such situations. Apache Hudi supports two write operation, namely "savepoint" and "restore" to assist you in this.
+
+## Savepoint
+
+As the name suggest, "savepoint" saves the table as of the commit time, so that it lets you restore the table to this 
+savepoint at a later point in time if need be. Users can run some validations on few candidate commits and trigger a savepoint 
+as applicable. Care is taken to ensure cleaner will not clean up any files that are savepointed. On similar lines, 
+savepoint cannot be triggered on a commit that is already cleaned up. In simpler terms, this is synonymous to taking a backup, 
+just that we don't make a new copy of the table, but just save the state of the table elegantly so that we can restore later 
+when in need. 
+
+## Restore
+
+This operation lets you restore your table to one of the savepointed commit. This is a destructive operation and so care 
```

Comment:
Can we say, "this operation cannot be undone" (or "reversed"), instead of "destructive"? Unless that is an industry term.